<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="bg">
      <property name="styleSheet">
       <string notr="true">QFrame{
	background-color:rgb(56, 58, 89);
	color:rgb(220, 220, 220);
	border-radius:20px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>25</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>25</number>
         </property>
         <property name="spacing">
          <number>20</number>
         </property>
         <item row="2" column="0">
          <widget class="QPushButton" name="btnMissile">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>520</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	background-color:rgb(254, 121, 199);;
	color:rgba(241,250,255,255);
	border-radius:10px;
	font-size:20px;
	font-family:sans-serif;
}

QPushButton:hover{
	color:rgba(255,255,255,255);
	border-radius:15px;
	font-size:20px;
	font-family:sans-serif;
	cursor:pointer;
}
</string>
           </property>
           <property name="text">
            <string>Activate Missile Tracking System</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/icons8_shutdown_96px.png</normaloff>icons/icons8_shutdown_96px.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QFrame" name="frame">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>120</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>800</width>
             <height>120</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="btnLink">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>120</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>800</width>
                <height>120</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:transparent;</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>icons/icons8-business-group-90.png</normaloff>icons/icons8-business-group-90.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>120</width>
                <height>120</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="btnLogout">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	background-color:rgb(254, 121, 199);;
	color:rgba(241,250,255,255);
	border-radius:10px;
	font-size:20px;
	font-family:sans-serif;
}

QPushButton:hover{
	color:rgba(255,255,255,255);
	border-radius:15px;
	font-size:20px;
	font-family:sans-serif;
	cursor:pointer;
}
</string>
           </property>
           <property name="text">
            <string>Log Out</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/icons8_close_pane_96px.png</normaloff>icons/icons8_close_pane_96px.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>55</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size:45px;
color:rgb(254, 121, 199);</string>
           </property>
           <property name="text">
            <string>Missile Tracking</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
